pub_sub/java/sdk/README.md

```diff
@@ -0,0 +1,61 @@
+# Dapr pub/sub
+
+In this quickstart, you'll create a publisher microservice and a subscriber microservice to demonstrate how Dapr enables a publish-subcribe pattern. The publisher will generate messages of a specific topic, while subscribers will listen for messages of specific topics. See [Why Pub-Sub](#why-pub-sub) to understand when this pattern might be a good choice for your software architecture.
```

Comment:
```suggestion
 "In this quickstart, there is a publisher microservice `checkout` and a subscriber microservice `order-processor` to demonstrate how Dapr enables a publish-subscribe pattern. `checkout` generates messages and publishes to a specific orders topic, and `order-processor` subscribers listen for messages of topic orders. "
```
Let's snap to a consistent README format speaking the present tense vs. future tense (checkout generates vs checkout will generate).  Also we either should delete teh Why Pub-Sub or make sure it links to a real topic.  We do have the [Pub-sub building block overview](https://docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-overview/).  Is that what we wanted?